### DISCUSSION TITLE
zh-hant/chapter_data_structure/classification_of_data_structure/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1566] [Announcements]
# zh-hant/chapter_data_structure/classification_of_data_structure/

動畫圖解、一鍵執行的資料結構與演算法教程

https://www.hello-algo.com/zh-hant/chapter_data_structure/classification_of_data_structure/

**Comment by Wade0325** [comment]
同一個節點的"節點值"與"節點指標"，有可能儲存在不連續的記憶體上嗎?
例如:
O    O     O   節點a     O        O      節點值a
O  節點b  O     O        O    節點值b
